skiko/buildSrc/src/main/kotlin/WasmCrossCompileTask.kt

```diff
@@ -0,0 +1,104 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.*
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.*
+import org.gradle.process.ExecOperations
+
+import javax.inject.Inject
+
+import java.io.File
+import java.util.LinkedHashSet
+
+abstract class WasmCrossCompileTask : DefaultTask() {
```

Comment:
Would be interesting to see split between generic cross-compilation task and wasm-specifics, but we could do that in follow up PRs.